src/fielddef.ts

```diff
@@ -100,6 +100,8 @@ export interface FieldDefBase<F> {
   /**
    * __Required.__ A string defining the name of the field from which to pull a data value
    * or an object defining iterated values from the [`repeat`](repeat.html) operator.
+   * Make sure you use proper escape characters as needed. `.`, `[`, and ']' have to
```

Comment:
I think it's worth saying 
```
__Note:__  Dots (`.`) and brackets (`[` and `]`) can be used to access nested objects.  
To access field names with dots or brackets, make sure to escape them using `\\`. 
For example, use `"field": "a\\.b` for a field named "a.b".
```

(Preview)

__Note:__ Dots (`.`) and brackets (`[` and `]`) can be used to access nested objects.  
To access field names with dots or brackets, make sure to escape them using `\\`. 
For example, use `"field": "a\\.b` for a field named "a.b".